app/scripts/login/change_password.html

```diff
@@ -38,17 +38,15 @@ <h3 translate>Eigenes Passwort wechseln</h3>
               </div>
             </div>
             <div class="container" ng-show="status === 'otpTwoFactor'">
-              <div class="alert alert-info" role="alert"><span translate>Bitte geben Sie das Einmalpasswort aus Ihrer
-                  Zwei-Faktor-Authentifizierungs-App ein.</span></div>
+              <div class="alert alert-info" role="alert"><span translate>Bitte geben Sie das Einmalpasswort aus Ihrer Zwei-Faktor-Authentifizierungs-App ein.</span></div>
               <div class="form-group">
                 <input type="text" class="form-control" placeholder="{{'Code' | translate}}"
                   ng-required="status === 'otpTwoFactor'" auto-focus ng-model="changePwd.secondFactorAuth.code">
               </div>
             </div>
   
             <div class="container" ng-show="status === 'emailTwoFactor'">
-              <div class="alert alert-info" role="alert"><span translate>Es wurde ein Zugangscode an ihre
-                  Email-Adresse</span> <b>{{person.email}}</b> <span translate>zugestellt.</span></div>
+              <div class="alert alert-info" role="alert"><span translate>Es wurde ein Zugangscode an ihre Email-Adresse</span> <b>{{person.email}}</b> <span translate>zugestellt.</span></div>
```

Comment:
Doing this we get a few somewhat teared apart strings to translate with no context. Wouldn't it be better to translate the whole string with the placeholder inside?

Reply:
why not. I just removed the "newline" characters because they are not needed and I find them confusing. I can add everything on a single translation string